src/framework/enhancement/js/ContextAwareness.js

```diff
@@ -204,15 +204,18 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
         }
     });
 
-    // Context awareness for the platform (operating system)
+    // Context awareness for the reported browser platform name (operating system)
 
-    fluid.contextAware.getPlatformName = function () {
-        return navigator.platform ? navigator.platform : undefined;
+    fluid.contextAware.getBrowserPlatformName = function () {
+        // Gets platform name in browser contexts
+        if (typeof(navigator) !== "undefined" && navigator) {
+            return navigator.platform ? navigator.platform : undefined;
+        }
     };
 
     fluid.contextAware.makeChecks({
-        "fluid.platformName": {
-            funcName: "fluid.contextAware.getPlatformName"
+        "fluid.browserPlatformName": {
+            funcName: "fluid.contextAware.getBrowserPlatformName"
```

Comment:
Can we make use of the `fluid.browser` namespace instead? So you'd end up with with `fluid.browser.platformName` and `fluid.contextAware.browser.getPlatformName` and similar throughout.